src/chrome/content/rules/xkcd.xml

```diff
@@ -88,15 +88,22 @@
 
         <exclusion pattern="^http://(www\.|m\.)?xkcd\.com/1663/" />
 
+		<test url="http://xkcd.com/1663/" />
+		<test url="http://m.xkcd.com/1663/" />
+		<test url="http://www.xkcd.com/1663/" />
 
-        <test url="http://xkcd.com/1663/" />
-        <test url="http://m.xkcd.com/1663/" />
-        <test url="http://www.xkcd.com/1663/" />
+	<exclusion pattern="^http://c\.xkcd\.com/" />
 
 
 	<rule from="^http://(www\.|m\.)?xkcd\.org/"
 		to="https://$1xkcd.com/" />
 
+	<rule from="^https://c\.xkcd\.com/"
+		to="http://c.xkcd.com/"
+		downgrade="1" />
+
+		<test url="https://c.xkcd.com/" />
+
```

Comment:
Why do an active downgrade rather than just excluding `c.xkcd.com` from the ruleset?  I'm getting a valid SHA-1 signed cert, which is problematic but better than plain http.